packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -273,6 +274,7 @@ const App = (props) => {
 
     return (
         <Box className="sf-app" {...styles.container}>
+            AppOrigin {appOrigin}
```

Comment:
Reminder to remove this line before merging the PR.

Reply:
is this deployed on MRT? can you share a link?

Reply:
Please see the PR description.